### PR TITLE
(IAC-626) Updating DHCP items for local_storage

### PR DESCRIPTION
**Pull request by thpang**
Added in code to adjust disk for DHCP hosts. This code was not in the initial push.

### CODE REVIEW

**Comment by thpang** [comment]
Looking to add this change in before getting the update to JK in a new docker image. Please review. The big thing to look at is the added disk in the dhcp vm host. Thanks.

File: templates/ansible/inventory.tmpl [hunk 1]
```diff
@@ -85,6 +85,20 @@ cr_server
 %{ endif ~}
 %{ endif ~}
 
+%{ if length(cr_ip) != 0 ~}
```
Comment (jarpat) [inline templates/ansible/inventory.tmpl:88]: This is already on L72-L86, don't think this is needed.

**Comment by thpang** [comment]
The duplicated `cr` block has been removed.

**Review by jarpat — APPROVED** [review]

**Comment by thpang** [comment]
Still looking for one more person to approve this one. Need to get this into the staging branch and cut a new docker image for JK. Thanks in advance.

**Review by riragh — APPROVED** [review]

**Pull request merged**